package unit

import "testing"

func TestAdd2(t *testing.T) {
	//t.Error("出错了")
}
